README.md

```diff
@@ -31,6 +31,72 @@ To start MinIO-Operator with default configuration, use the `minio-operator.yaml
 kubectl apply -f https://raw.githubusercontent.com/minio/minio-operator/master/minio-operator.yaml
 ```
 
+MinIO Operator relies on [MinIO Disco](https://github.com/minio/disco) for Service Discovery and so that applications inside the cluster can discover instances if they lookup an instance by bucket (i.e: `bucket1.tenant.minio.local`), so you need to forward any request for the domain `minio.local` on you cluster DNS. 
```

Comment:
This implies that tenant name needs to be unique across all namespaces. Would we need to add a corresponding check in m3 or in the operator itself? This is with the model the CRD lives in the namespace of the minio instance.

Reply:
You are right, I’ll update it so it includes the namespace in the route

Reply:
done